.gitignore

```diff
@@ -258,3 +258,6 @@ paket-files/
 *.log
 *.cso
 *.zip
+
+# Mac Files
+.DS_Store
```

Comment:
any reason this is still included in the root .gitignore?

Reply:
Yes -- if you pull the repo on a mac, it adds .DS_Store files to all of the directories (not just the SwiftClient), so anyone looking to make any changes via a Mac needs to ignore this file for clarity